GmsCore/Android.mk

```diff
@@ -32,5 +32,6 @@ LOCAL_OVERRIDES_PACKAGES := com.qualcomm.location
 LOCAL_REQUIRED_MODULES := privapp-permissions-com.google.android.gms.xml default-permissions-com.google.android.gms.xml sysconfig-com.google.android.gms.xml
 LOCAL_PRODUCT_MODULE := true
 LOCAL_USES_LIBRARIES := com.android.location.provider
+LOCAL_OPTIONAL_USES_LIBRARIES := androidx.window.extensions androidx.window.sidecar
```

Comment:
On my local builds, running LineageOS 18.1, I also had to extend the privapp-permissions-com.google.android.gms.xml file and add this permission:
<permission name="android.permission.MANAGE_USB"/>

Without it the OS wouldn't boot because its a privileged permission and its required by FIDO, added recently to microg. 

Have you also experienced this?

Reply:
Interesting, because I have not added that yet but my 19.1 builds on google devices are booting without it. Looks like it was recently added [here](https://github.com/FriendlyNeighborhoodShane/MinMicroG/commit/a1942c948bd37af813d2f7302b11ab7dbe48db0b) also, which I have used as a point of reference in the past when these permissions were last updated... 

Thanks for mentioning it.

Reply:
@mikalackis that was really helpful, thanks again. Adding the missing permission just resulted in a working build of 20.0 (on Samsung S7) for @Exodusnick using the docker-lineage-cicd project. We are stoked!

Reply:
Glad to help, looking good with that 20 on S7 haha :)